auto_nag/scripts/phonebook.py

```diff
@@ -35,11 +35,27 @@
 
 
 class PhonebookDirectory():
-
-    def __init__(self, config=CONFIG_JSON):
-        config = json.load(open(config, 'r'))
+    def __init__(self, TEST=False):
         print "Fetching people from phonebook..."
-        self.people = json.loads(requests.get(PEOPLE_URL, auth=(config['ldap_username'], config['ldap_password'])).content)
+        if TEST:
```

Comment:
yes, the sample manager/people data should be stored in a json file in the test directory.